File: test/front-end/src/app/model/export-bill/exportBill.ts
import {Exportbilltype} from './exportbilltype';
import {Employee} from './employee';
import {Manufacturer} from './manufacturer';

export interface ExportBill {
  exportBillId?:number;
  exportBillCode?: string;
  exportBillDate?: string;
  exportBillReason ?: string;
  exportBillAddress?: string;
  exportBillType?: Exportbilltype;
  employee?: Employee;
  manufacturer?: Manufacturer;
}

File: test/back-end/src/main/java/com/example/demo/model/entity/export_bill/ExportBill.java
package com.example.demo.model.entity.export_bill;

import com.example.demo.model.entity.employee.Employee;
import com.example.demo.model.entity.manufacturer.Manufacturer;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "export_bill")
public class ExportBill {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long exportBillId;
    private String exportBillCode;
    private String exportBillDate;
    private String exportBillReason;
    private String exportBillAddress;
    private Boolean flag=true;
    @ManyToOne
    @JoinColumn(name = "export_bill_type_id")
    private ExportBillType exportBillType;
    @ManyToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "employee_id")
    private Employee employee;
    @ManyToOne
    @JoinColumn(name = "manufacturer_id")
    private Manufacturer manufacturer;
    @OneToMany(mappedBy = "exportBill")
    @JsonIgnore
    private List<ExportBillDetail> exportBillDetails;



}